### PR TITLE
Refactors homepage css

### PR DESCRIPTION
**Pull request by simonschwartz**
The PR closes SITES-451

Refactors homepage css
Updates copy
Replaces 'About Australia' images with HTML

text

### CODE REVIEW

File: app/assets/stylesheets/_homepage.scss [hunk 1]
```diff
@@ -1,143 +1,196 @@
...
-
-
   }
+}
 
-  .home-heading {
-    border-bottom: 6px solid $light-aqua;
-    padding-bottom: 1.6rem;
-  }
+/*
```
Comment (maxious) [inline app/assets/stylesheets/_homepage.scss]: Do we need this commented out code?

File: app/views/templates/root_node.html.haml [hunk 2]
```diff
@@ -13,45 +14,49 @@
...
-            %span= @agencies.count
-            Agencies
-        %li
-          %a{href: '/ministers'}
-            %span= @ministers.count
-            Ministers
+          .meta= news.release_date
+  =link_to 'All news', news_articles_path, :class => "see-more"
+
+%section.about-governemnt
```
Comment (tessereth) [inline app/views/templates/root_node.html.haml]: That's not how you spell government

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,6 +7,8 @@ This file is influenced by http://keepachangelog.com/.
 
 - Support setting user names in admin
 - Allow users to specify their name when they sign up
+- Refactor homepage images into responsive HTML components
+- Update homepage copy
```
Comment (tessereth) [inline CHANGELOG.md]: These things are changes not additions, aren't they?

**Comment by ltankey** [comment]
LGTM

**Comment by maxious** [comment]
LGTM

**Comment by tessereth** [comment]
LGTM

**Comment by simonschwartz** [comment]
LGTM

**Pull request merged**